specification/automation/resource-manager/Microsoft.Automation/2015-10-31/account.json

```diff
@@ -197,7 +212,12 @@
         },
         "parameters": [
           {
-            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+            "name": "resourceGroupName",
```

Comment:
I'm surprised that this change is introducing more code duplication. Any particular reason behind this?

Reply:
Agree, but unfortunately when generating documentation, parameters defined this was were not shown in parameters table in docs. Looking at other working examples that work suggested this change.